src/test/java/org/cloudfoundry/autosleep/dao/repositories/RouteBindingRepositoryTest.java

```diff
@@ -0,0 +1,147 @@
+package org.cloudfoundry.autosleep.dao.repositories;
```

Comment:
Duly noted. This was actually only a copy/paste from the ApplicationBindingRepositoryTest.java, explaining why it's formatted "the old way"